src/networks/starknet/constants.ts

```diff
@@ -393,7 +341,89 @@ export function createConstants(networkId: NetworkID) {
           }
         }
       }
-    }
+    },
+    ...(config.Strategies.EVMSlotValue
+      ? [
+          {
+            address: config.Strategies.EVMSlotValue,
+            name: 'EVM slot value',
+            about:
+              'A strategy that allows to use the value of an slot on EVM chain (for example ERC-20 balance on L1) as voting power.',
+            icon: IHCode,
+            generateSummary: (params: Record<string, any>) =>
+              `(${shorten(params.contractAddress)}, ${params.slotIndex})`,
+            generateParams: (params: Record<string, any>) => {
+              return CallData.compile({
+                contract_address: params.contractAddress,
+                slot_index: uint256.bnToUint256(params.slotIndex)
+              });
+            },
+            generateMetadata: async (params: Record<string, any>) => ({
+              name: 'EVM slot value',
+              properties: {
+                symbol: params.symbol,
+                decimals: parseInt(params.decimals),
+                token: params.contractAddress,
+                payload: JSON.stringify({
+                  chainId: params.chainId,
+                  contractAddress: params.contractAddress,
+                  slotIndex: params.slotIndex
+                })
+              }
+            }),
+            parseParams: async (params: string, metadata: StrategyParsedMetadata | null) => {
+              if (!metadata || !metadata.payload) throw new Error('Missing metadata');
+
+              const [contractAddress] = params.split(',');
+              const { chainId, slotIndex } = JSON.parse(metadata.payload);
+
+              return {
+                contractAddress,
+                decimals: metadata.decimals,
+                symbol: metadata.symbol,
+                slotIndex,
+                chainId
+              };
+            },
+            paramsDefinition: {
+              type: 'object',
+              title: 'Params',
+              additionalProperties: false,
+              required: ['contractAddress', 'slotIndex', 'chainId'],
+              properties: {
+                contractAddress: {
+                  type: 'string',
+                  format: 'address',
+                  title: 'Token address',
```

Comment:
This can be used on a contract that is not a token, I would rather use this:
```suggestion
                  title: 'Contract address',
```